bin/lib/aws/translation-cache-bucket.js

```diff
@@ -0,0 +1,75 @@
+const AWS = require('aws-sdk');
+
+const region = process.env.AWS_REGION;
+const cacheBucket = process.env.AWS_CACHE_BUCKET;
+
+const S3 = new AWS.S3();
```

Comment:
I usually try and name my variables for specific AWS products to be more generic like 'fileStore' just so if we ever used something else it makes it slightly easier to switch out.